lib/utils/angular/properties/properties.dart

```diff
@@ -68,12 +51,39 @@ bool getBool(inputValue) {
       inputValue, 'inputValue', 'Expected a String, or bool type');
 }
 
+/// Parses html attribute [String] to a [bool].
+///
+/// Should be used to parse values passed to @Attribute constructor argument.
+///
+/// This does not fully follow the HTML boolean attribute definition
```

Comment:
> This does not fully follow the HTML boolean attribute definition (https://stackoverflow.com/a/4139805)

"Does not fully follow" would mean that it supports a _subset_ of the definition, but that in theory it could be extended/completed so that it _does_ follow the full definition. This is not the case.

IMHO, this function is in **conflict** with the HTML boolean attribute definition, since it matches the HTML boolean attribute definition _only_ for the case where the attribute value is an empty string, and it goes against the definition for other cases (where the value is "true" or "false", and it fails to recognize the case where the value matches the attribute name, ignoring case).

To make this clear, maybe this function should be named `ngAttributeToBool()` to be in contrast with what might be named `htmlAttributeToBool()`.

cc @kwalrath @matanlurey

Reply:
@rkj Any thoughts on rewording the comments here for more clarity? We can update them in a future sync to github.